pyclam/manifold.py

```diff
@@ -107,6 +107,11 @@ def depth(self) -> int:
         """ The depth in the tree at which the cluster exists. """
         return len(self.name)
 
+    @property
+    def parent(self) -> 'Cluster':
+        """ Returns the parent of self """
+        return self.manifold.select(self.name[:-1])
```

Comment:
Should we just store a parent reference?

Reply:
We are not using this anymore. I took it out